src/main.rs

```diff
@@ -82,20 +82,20 @@ fn run(options: Options) -> Result<(), anyhow::Error> {
             let (contents, chunk_name) = load_script(&script)?;
             let lua = Lua::new();
 
-            lua.context(move |context| {
+            {
                 let lua_args = args
                     .into_iter()
-                    .map(|value| value.to_lua(context))
+                    .map(|value| value.to_lua(&lua))
                     .collect::<Result<Vec<_>, _>>()?;
 
-                RemodelContext::new(auth_cookie).inject(context)?;
+                RemodelContext::new(auth_cookie).inject(&lua)?;
 
-                RemodelApi::inject(context)?;
-                RobloxApi::inject(context)?;
+                RemodelApi::inject(&lua)?;
+                RobloxApi::inject(&lua)?;
 
-                let chunk = context.load(&contents).set_name(&chunk_name)?;
+                let chunk = lua.load(&contents).set_name(&chunk_name)?;
                 chunk.call(MultiValue::from_vec(lua_args))
-            })?;
+            }?;
```

Comment:
Instead of creating a new scope here and using `?`, let's just stick `?` onto the `chunk.call` line. I like that the mlua version of this code is simpler.